cmd/translate/main_grpc_integration_test.go

```diff
@@ -157,7 +160,8 @@ func createService(ctx context.Context, t *testing.T) *translatev1.Service {
 func Test_UploadTranslationFile_gRPC(t *testing.T) {
 	t.Parallel()
 
-	ctx := context.Background()
+	ctx, span := otel.Tracer(name).Start(context.Background(), t.Name())
+	defer span.End()
```

Comment:
It would be nice to mark the span as failed when the test fails to improve visibility.

```suggestion
func trace(ctx, t) (ctx, func()) {
  ctx, span := tracer.Start(ctx, t.name)
  
  return func() {
    if t.Failed() {
      span.SetStatus...
      span.RecordError...
    }
    
    span.End()
  }
}
	ctx, spanEnd := trace(ctx, t)
	defer spanEnd()
```